charts/argo-cd/templates/argocd-applicationset/role.yaml

```diff
@@ -48,40 +48,31 @@ rules:
     - ""
     resources:
     - secrets
-    - configmaps
-    verbs:
-    - get
-    - list
-    - watch
-  - apiGroups:
-    - apps
-    - extensions
-    resources:
-    - deployments
     verbs:
     - get
     - list
     - watch
-  # Leader election
   - apiGroups:
     - ""
+    - coordination.k8s.io
     resources:
     - configmaps
+    - leases
```

Comment:
Removed extra section as both configmaps and leases have same rbac.

Reply:
I dont think think this is a good idea.
Why should they be the same (esp. the verbs)?

Reply:
Check the verbs for configmaps and leases when you put them side-by-side.

Reply:
I've kept it separated even when verbs are the same. Just joined configmap groups.